src/common/hotbar-types.ts

```diff
@@ -33,14 +33,18 @@ export interface HotbarItem {
   }
 }
 
-export type Hotbar = Required<HotbarCreateOptions>;
+export type Hotbar = Required<CreateHotbarData>;
```

Comment:
Do we really need utility type `Required` here? If yes, name types to make the reasoning more apparent.

Reply:
What you would like instead? Spelling out the field manually?